trailblazer/store/filters/analyses_filters.py

```diff
@@ -97,12 +103,13 @@ def apply_analysis_filter(
     filter_functions: list[Callable],
     analysis_id: int | None = None,
     case_id: str | None = None,
-    search_term: str | None = None,
     comment: str | None = None,
+    order_id: int | None = None,
+    priorities: list[str] | None = None,
```

Comment:
I think the endpoint allows submitting multiple priorities, hence the plural. So I think I prefer it the way it is. Either way, the field is only changed so that the parameters are in alphabetical order and I will keep it as is for now.